### DISCUSSION TITLE
Newbie questions

### DISCUSSION THREAD

**Discussion by orso7** [rsjaffe/MIDI2LR#787] [Q&A]
I am thinking about getting a midi controller over the Loupdeck+ but I am a bit unsure what is possible with Midi2LR an different midi controllers.
The three I stumbled upon were:
Behringer X-Touch mini (ofc, this is the favorite but I dislike the form factor somehow )
Behringer CMD DC-1
Akai MPD218

As I understand the x-Touch mini has two hardware layers which means I have double the controls inside each of the profiles (lets say development), but is it also possible with the CMD DC-1 wich has no hardware layers to put another software layer inside the development profile? so like the hw-layers but through MIDI2LR? Is there a way to add hw layers somehow, with something like adding a arduino + a switch?

Does anyone have a MPD218 or CMD DC-1 in use? I can't find out for sure if the knobs do have a button included or are just rotating.

Are there any MIDI controlers that are a bit more minimalistic? I would love just one big knob (that feels super nice) and a ton of buttons surrounding it. Was thinking about building one myself but maybe something like it already exists.

EDIT: And one last question, does anyone have found a cheap solution to get all those controllers wireless?

**Comment by remov-b4-flight** [comment]
Hi, olso7,
I 'm developing MIDI controller DIY,[LrTMAX](https://github.com/remov-b4-flight/LrTMAX) repository
Watching in for X and other SNS, No project like you told as very simple way. 
Of course you can make your own MIDI controller that you want.
Many hobbist their own controllers on Arduino board. You can learn from their projects.
(My project is filly scratch build on STM32 board)
